### PR TITLE
Sleob fix

### PR DESCRIPTION
**Pull request by stephenwaite**

### CODE REVIEW

File: interface/billing/sl_eob_invoice.php [hunk 1]
```diff
@@ -248,10 +248,12 @@ function updateFields(payField, adjField, balField, coPayField, isFirstProcCode)
 
               // Handle deletes. row_delete() is borrowed from deleter.php.
             if ($ALLOW_DELETE && !$debug) {
-                foreach ($_POST['form_del'] as $arseq => $dummy) {
-                    row_delete("ar_activity", "pid = '$patient_id' AND " .
-                      "encounter = '$encounter_id' AND sequence_no = '$arseq'");
-                }
+                if (is_array($_POST['form_del']) {
```
Comment (bradymiller) [inline interface/billing/sl_eob_invoice.php]: missing a trailing parenthesis :)
``````
if (is_array($_POST['form_del']) {

**Comment by bradymiller** [comment]
some good errors to address are the deprecated ones. click on the 'Details' above to see Travis CI build fails. Then click on 7.1 lint_syntax row. And will see output and would be good to address the deprecated ones.

**Comment by bradymiller** [comment]
btw, my suggestion above is just if your interested. Cleaning up warnings in specific scripts (like billing as you are doing) is also a good strategy (ie. anything that fixes the code is good :) ).

**Comment by stephenwaite** [comment]
be glad to look into the deprecated ones, thanks Brady

**Comment by stephenwaite** [comment]
can't find the bug why the travis ci failed and clicking on the details just states the file where the error is

**Comment by bradymiller** [comment]
You have failed the PSR2 styling check. Would be nice for that report to list the actual crime ( @TheToolbox , is this possible? )

This is why we are going to convert you to Php Storm :)
http://www.open-emr.org/wiki/index.php/OpenEMR_PHPStorm_Setup

I'll check it out this evening in my editor (will fix it up and bring it into the codebase unless somebody beats me to it).

**Comment by stephenwaite** [comment]
i'd like to be that somebody :)
but when i go to phpstorm and try to follow the above instructions, i can't choose custom in the coding standard dropdown, guessing because i don't have php code sniffer installed as the above box in php storm states

**Comment by bradymiller** [comment]
@stephenwaite ,

I think we need to update the wiki page a bit there. According to @juggernautsei , need to do following first to allow to choose the custom:
https://confluence.jetbrains.com/display/PhpStorm/PHP+Code+Sniffer+in+PhpStorm#PHPCodeSnifferinPhpStorm-Prerequisites

I did a poor man route where i git cloned the https://github.com/squizlabs/PHP_CodeSniffer repo and directed phpstorm to the binary in there.

-brady

**Comment by stephenwaite** [comment]
cool, thanks, got it to choose custom, i'll work on the style tomorrow

**Comment by bradymiller** [comment]
nice,

It's pretty neat. It will highlight the text where it is not following PRS2 standard there along with the problem(in cobalt theme, makes a squiggly red underline at the error). There are also red little bars at the right scrollbar at each PSR2 error. 

Note that we recommend to use phpcs_strict.xml for php storm sniffing, which will identify all PSR2 issues. However, do note the code base check is less stringent since there are some rules that will take more effort to support, which is the phpcs.xml item.(this means you do not need to fix all the PSR2 issues, just the ones that would be identified by phpcs.xml; so if get confused what fixes are mandatory, can change from phpcs_strict.xml to phpcs.xml in phpstorm)

**Comment by TheToolbox** [comment]
W.R.T. listing off the errors in CI, it is possible and was my original intent. I believe when I updated from phpcs 2.7 to 3.0, the syntax for multiple reports may have changed. I'll try to throw together a pull request tonight to add it back in!

**Comment by TheToolbox** [comment]
Should be improved in #969

**Comment by bradymiller** [comment]
very cool!
After bringing in @TheToolbox improvements into the codebase I had travis rebuild this PR code. And you can now see the error and line number of the error.

**Comment by stephenwaite** [comment]
thanks @bradymiller @TheToolbox , i'll keep working with phpstorm to see if i can see the errors there too

**Comment by bradymiller** [comment]
will bring this into the codebase. thanks for the fix! -brady

**Pull request merged**